src/amo/components/HomeHeroBanner/index.js

```diff
@@ -1,19 +1,49 @@
 /* @flow */
 import * as React from 'react';
 import { compose } from 'redux';
+import makeClassName from 'classnames';
 
 import translate from 'core/i18n/translate';
 import Hero from 'ui/components/Hero';
 import HeroSection from 'ui/components/HeroSection';
 import type { I18nType } from 'core/types/i18n';
+import tracking from 'core/tracking';
+import { withExperiment } from 'core/withExperiment';
 
 import './styles.scss';
 
 type Props = {|
   i18n: I18nType,
+  trackExperimentClick: () => void,
+  variant: string,
 |};
 
-export class HomeHeroBannerBase extends React.Component<Props> {
+type InternalProps = {|
+  ...Props,
+  _tracking: typeof tracking,
+|};
+
+export const AB_HOME_HERO_TEST_NAME = 'HOME_HERO';
+export const AB_HOME_HERO_VARIANT_A = 'ScaledDown';
+export const AB_HOME_HERO_VARIANT_B = 'Large';
+
+export class HomeHeroBannerBase extends React.Component<InternalProps> {
+  static defaultProps = {
+    _tracking: tracking,
+  };
+
+  componentDidMount() {
+    const { _tracking, variant } = this.props;
+
+    if (variant) {
+      _tracking.sendEvent({
+        action: `${AB_HOME_HERO_TEST_NAME} Page View`,
+        category: `AMO ${AB_HOME_HERO_TEST_NAME}_EXPERIMENT: ${variant}`,
+        label: '',
+      });
+    }
```

Comment:
Do we actually want to send page view events?

Reply:
Well what if they never click on a hero? how we will know which version they are looking at?

Reply:
we compare clicks between the two variants?

Reply:
hey @willdurand, this is what I am referring to: https://github.com/mozilla/addons-frontend/issues/6018#issuecomment-418814678

Reply:
oh shoot, right.. we also want to measure bounce rate.

Reply:
sorry, yes so this is needed. we want to test both engagement (clicking on links) and bounce rate (users stay on the site instead of abandoning it)

Reply:
so how I have this then is ok?

Reply:
nope, bounce rate is computed by google analytics. we will know which home gets the best rate thanks to the "page view" event you send